src/spec.ts

```diff
@@ -255,60 +254,50 @@ function normalizeRangedUnit(spec: UnitSpec) {
 }
 
 
-// FIXME(#1804): rewrite this
+// FIXME(#1804): re-design this
```

Comment:
Can you elaborate on what you think is wrong with this. A fixme is a strong comment.